### DISCUSSION TITLE
Welcome to mine-city-2000 Discussions!

### DISCUSSION THREAD

**Discussion by jgosar** [jgosar/mine-city-2000#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Mu0n** [comment]
I couldn't compile it under Visual Studio 2010 C# Express Edition.
I get these errors:
Error	1	Identifier expected; 'static' is a keyword	D:\mine-city-2000\MineCity2000\SCMapper.cs	5	7	MineCity2000
Error	2	Expected class, delegate, enum, interface, or struct	D:\mine-city-2000\MineCity2000\SCMapper.cs	5	14	MineCity2000

**Reply by jgosar** [reply]
Hi, it seems that this feature has been introduced in a later version of the .NET framework than you are using. See https://stackoverflow.com/questions/44535491/using-static-throwing-error

Please try with Visual Studio 2019.
